### ISSUE TITLE
IGSN/Sesar login validation broken

### ISSUE THREAD

**Issue by mandrewj** [BioKIC/NEON-Biorepository#324]
When trying to mint and assign IGSN's through the NEON management tool, authentication with the SESAR system seems to fail.

To replicate, go to https://biorepo.neonscience.org/portal/neon/igsncontrol.php and select any collection that needs IGSNs to be assigned. Then use sesar login to go to next page.

I continue to get the following "Validation call failed" error, though I am able to log into my sesar account with the same credentials at https://www.geosamples.org/
<img width="1394" alt="Screen Shot 2022-09-13 at 12 25 08 PM" src="https://user-images.githubusercontent.com/6679254/189991871-a4169324-ff48-4469-942d-0c9d710b319e.png">

This could be due to an update to sesar's API and login handling?

**Comment by egbot** [comment]
This seems to be working now. I'm able to login fine with my credentials and harvest IGSNs/ I know that they were preforming updates for a short period on Sept 8th. Perhaps, it took longer than planned to restore certain functions? Or it was just a glitch. Closing issue, though reopen if you are still experiencing issues.

**Comment by mandrewj** [comment]
Reopening this issue - still won't let me validate even after reseting my password.  I've reconfirmed that I can log into the SESAR system directly, just cannot authenticate to assign IGSNs through the biorepo portal.  Checked with Kelsey and Laura RP who don't have accounts or ability to log in themselves.

Could have something to do with Ed owning the SESAR namespace and it only being shared with me?

**Comment by mandrewj** [comment]
We were able to get this working. 

SESAR returned an empty string for my login credentials when given the correct user/login. When an incorrect password was given, SESAR returned an error - this indicated that the login was registered but the return string was not what SESAR's documentation said it should return.

When Ed would send his credentials to SESAR, it returned a valid response and listed his IGSN codes registered to him, and when an incorrect password was given he received the same error code as I did.

I went into SESAR and added a fake namespace (code) to my account so that I now both "own" an IGSN code and have access to the 'NEO' namespace.  My login credentials now return an accepted response listing what I have access to.

It is not clear why this is happening, but our guess is that users need to own their own namespace and to create a user-specific code in order for their API service to return a string at all.  In the future, other users should create a code to get access to the API credential validation services.
